src/parser/core/modules/Interrupts.tsx

```diff
@@ -59,10 +59,18 @@ export abstract class Interrupts extends Module {
 	 * @param missedCasts
 	 * @param missedTime
 	 */
-	private suggestionWhy(missedCasts: CastEvent[], missedTime: number): JSX.Element {
+	protected suggestionWhy(missedCasts: CastEvent[], missedTime: number): JSX.Element {
```

Comment:
Is making this overridable intended?  If so, can you add a doc comment indicating what consumers should provide?

Reply:
Yes, it's absolutely intended - it would make sense if someone wanted to change the why text to be more about missed cast time vs the number of missed casts (even though the default is to provide both). I could see samurai not caring about time at all (since the new Iaijutsu has such a short casting time), but only that you interrupted.

I'll absolutely add more docs to this, though.